<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RenderConfig</class>
 <widget class="QDockWidget" name="RenderConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rendering Options</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Skeleton (nodes and edges)</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Edge Radius</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Node Size</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="sbNodeSize">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimum">
          <double>0.010000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="sbEdgeRadius">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimum">
          <double>0.100000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="2" rowspan="2">
        <widget class="QCheckBox" name="cbLink">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Link</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Transparency</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Number of depth peels</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Mode</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="sbNumDepthPeels">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="cbTransparencyMode">
         <item>
          <property name="text">
           <string>Backfaces, Frontfaces</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Sort Triangles</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Depth Peeling</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Background</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Color</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="UI::ColorButton" name="btnBgColor" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbGradient">
         <property name="text">
          <string>Gradient</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UI::ColorButton</class>
   <extends>QWidget</extends>
   <header location="global">UI\colorbutton\colorbutton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>sbNodeSize</sender>
   <signal>valueChanged(double)</signal>
   <receiver>RenderConfig</receiver>
   <slot>OnNodeSizeChanged(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sbEdgeRadius</sender>
   <signal>valueChanged(double)</signal>
   <receiver>RenderConfig</receiver>
   <slot>OnEdgeRadiusChanged(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbTransparencyMode</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>RenderConfig</receiver>
   <slot>OnTransparencyModeChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>152</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sbNumDepthPeels</sender>
   <signal>valueChanged(int)</signal>
   <receiver>RenderConfig</receiver>
   <slot>OnNumDepthPeelsChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>152</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnNodeSizeChanged(double)</slot>
  <slot>OnEdgeRadiusChanged(double)</slot>
  <slot>OnTransparencyModeChanged(int)</slot>
  <slot>OnNumDepthPeelsChanged(int)</slot>
 </slots>
</ui>
